cmake/dqm4hep_settings.cmake

```diff
@@ -4,6 +4,9 @@ set( CMAKE_ALLOW_LOOSE_LOOP_CONSTRUCTS TRUE ) # default in cmake 2.6
 set( DQM4HEP_CMAKE_MODULES_ROOT ${CMAKE_CURRENT_LIST_DIR} )
 set( CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE )
 
+# Automatically tell CMake where to find DQM4HEPConfig.cmake
+set( CMAKE_PREFIX_PATH ${CMAKE_PREFIX_PATH} ${CMAKE_CURRENT_LIST_DIR} )
```

Comment:
Can you actually switch the order to:
```cmake
set( CMAKE_PREFIX_PATH ${CMAKE_CURRENT_LIST_DIR} ${CMAKE_PREFIX_PATH} )
```
to avoid potential conflicts ?

Reply:
Sure, can you elaborate on why the first version would have conflicts?